tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -84,6 +118,22 @@ def circuit(x=None):
         state = circuit.device.state.ravel()
         assert len(set(state[len(inpt) :])) == 1
 
+    @pytest.mark.parametrize("inpt", NOT_ENOUGH_BROADCASTED_FEATURES)
+    @pytest.mark.parametrize("pad", [complex(0.1, 0.1), 0.0, 1.0])
+    def test_prepares_padded_state_broadcasted(self, inpt, pad):
+        """Checks the state for real and complex padding constants."""
+
+        n_qubits = 2
+        dev = qml.device("default.qubit", wires=n_qubits)
+
+        @qml.qnode(dev)
+        def circuit(x=None):
+            qml.AmplitudeEmbedding(features=x, wires=range(n_qubits), pad_with=pad, normalize=False)
+            return qml.state()
+
+        state = circuit(x=inpt)
+        assert len(np.unique(state[:, inpt.shape[1] :])) == 1
```

Comment:
Might be worth describing how exactly this line helps check the padding constant usage.